app/schemas/index.js

```diff
@@ -29,8 +29,8 @@ export const outboundMessageSchema = joi.object({
       .pattern(/^[a-z0-9-_]+$/)
       .required(),
     notifyTemplateId: joi.string().guid({ version: 'uuidv4' }).required(),
-    commsType: 'email',
-    commsAddresses: email.required(),
+    commsType: joi.string().valid('email').required(),
+    commsAddresses: email.required(), // Note: array has maxItems: 10
```

Comment:
Is this an array type? The comment says array but the definition of email on line 11 says string.

Reply:
Yes, is oneOf string or array. We only support a string currently. Just wanted to remind us when converting to array that the array seems to be capped to 10. Guessing this might not support our use case.